arm/Microsoft.Compute/galleries/images/.parameters/minimum_parameters.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Please remove. 
We don't have individual parameter files for child-resources but test them via the top-level parent.
The primary reason is, that they are not actually tested with the workflow and may run a high chance to become stale.